### PR TITLE
add time locked tokens

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

File: JumpscaleLibs/clients/stellar/balance.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        if response_balance['asset_type']=='native':
+            asset_code='XLM'
+            asset_issuer=None
+        else:
+            asset_code=response_balance['asset_code']
+            asset_issuer=response_balance['asset_issuer']
+        return Balance(balance,asset_code,asset_issuer)
+
+    def __str__(self):
+        return "{}".format(self.balance)
```
Comment (xmonader) [inline JumpscaleLibs/clients/stellar/balance.py]: return f"{self.balance}

File: JumpscaleLibs/clients/stellar/balance.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    
+    def __repr__(self):
+        return "{} {} ({})".format(self.balance,self.asset_code,self.asset_issuer)
```
Comment (xmonader) [inline JumpscaleLibs/clients/stellar/balance.py]: return f"{self.balance},{self.asset_code},{self.asset_issuer}"

File: JumpscaleLibs/clients/stellar/balance.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class Balance(object):
+
+    def __init__(self, balance =0.0, asset_code='XLM', asset_issuer=None):
```
Comment (xmonader) [inline JumpscaleLibs/clients/stellar/balance.py]: extra space in balance = 0.0

File: JumpscaleLibs/clients/stellar/StellarClient.py [hunk 4]
```diff
@@ -56,14 +57,18 @@ def _init(self, **kwargs):
...
+    def _get_horizon_server(self):
         return Server(horizon_url=_HORIZON_NETWORKS[str(self.network)])
 
     def get_balance(self):
         """Gets balance for address
         """
-        response= self._get_horizon_server().accounts().account_id(address).call()
-        return (response['balances'])
+        response= self._get_horizon_server().accounts().account_id(self.address).call()
+        balances=j.baseclasses.dict()
```
Comment (xmonader) [inline JumpscaleLibs/clients/stellar/StellarClient.py]: any specific reason to use this one?

Comment (DylanVerstraete) [reply JumpscaleLibs/clients/stellar/StellarClient.py]: No, which one should we use?

Comment (robvanmieghem) [reply JumpscaleLibs/clients/stellar/StellarClient.py]: It had a __str and __repr__ implementation  but ignore it for now, working on a specific balances object  that differentiates between locked, free and multisig balances and has the proper representation methods

**Review by robvanmieghem — APPROVED** [review]
Modified my remarks and added the ability to see and claim locked tokens

**Comment by robvanmieghem** [comment]
@xmonader can you recheck with the modified/added code?

**Pull request merged**